### ISSUE TITLE
Keep the gozim bar in the pages

### ISSUE THREAD

**Issue by viric** [akhenakh/gozim#23]
Visiting a zim page, takes out all gozim interface from the web page.

It'd be nice if it were still visible, for Search and so on.

**Comment by akhenakh** [comment]
The way the zim files are rendered makes it difficult, it returns a full <html></html>, I don't see any solutions out of ugly iframes.I'm not into html so except if you want to give it a try, I will close this one.

**Comment by kelson42** [comment]
With kiwix-serve, we introduce a bar by modifying the DOM. Not perfect solution, but works in 99% of the cases... Here is how it looks like: http://library.kiwix.org/wikipedia_en_wp1/

**Comment by akhenakh** [comment]
Looks good :+1: 

kiwix-serve will be maintained?
Cause the absence of working/maintained http server for zim files was the main reason to create gozim, if kiwix-serve is here to stay, gozim is not that important anymore (except kiwix-serve is using c++ :p )

**Comment by kelson42** [comment]
@akhenakh

Why do you think kiwix-serve is not maintained? kiwix-serve is an important part of Kiwix project, has been existing for ~ 5years and is maintained as good as we can by volunteers developer.

I really appreciate to have many other ZIM readers, and this is great to have one in go too. Would be sad to see you abandoning that project.

**Comment by akhenakh** [comment]
@kelson42 I find it hard to get the right information about binaries and Kiwix.
http://www.kiwix.org/wiki/Debian 
and latest binary is from 2014 but maybe I can't find the right one?

I've spent a lot of time looking for a simple working http server for zim file before creating gozim, I found some in C++ but it was a pain to compile mainly because it was using very old libraries, I don't remember if it was kiwix-serve, I'll give it a try.

I hope to continue but it's difficult to maintain this project, alone, behind the official implementation.

**Comment by akhenakh** [comment]
I've found the recent link for ARM, I'm testing it, works very well

**Comment by kelson42** [comment]
Yes, last binaries are from 2014. We should release 0.9.1 in the next months but this includes only a few fixes. Feel free to improve or report if you find things which are not clear on openzim.org (web site for the format). Happy to hear kiwix-serve works for you :)
